tests/unit/test_group_project.py

```diff
@@ -230,6 +230,30 @@ def test_peer_grade_questions(self, expected_result, *stage_questions):
 
             get_stages.assert_called_with(PeerReviewStage.CATEGORY)
 
+    @ddt.data(
+        (None, []),
+        (None, [None, None, None]),
+        ('2016-10-29 04:30:48', ['2016-02-26 04:30:48', None, '2016-10-29 04:30:48']),
+        ('2016-10-29 04:30:48', ['2016-02-26 04:30:48', None, '2016-10-29 04:30:48', None]),
+        ('2016-10-29 04:30:48', ['2016-02-26 04:30:48', '2016-10-29 04:30:48', '2016-02-29 04:30:48']),
+        ('2016-10-29 04:30:48', ['2016-10-29 04:30:48', '2016-02-26 04:30:48', '2016-02-29 04:30:48']),
+        ('2016-10-29 04:30:48', ['2016-02-26 04:30:48', '2016-02-29 04:30:48', '2016-10-29 04:30:48']),
+        ('2016-10-29 04:30:48', ['2016-02-26 04:30:48', '2016-10-29 04:30:48', '2016-10-29 04:30:48']),
+        ('2016-10-29 04:30:48', ['2016-02-26 04:30:48', '2016-10-29 04:30:48', '2016-02-26 04:30:48']),
+        ('2016-10-29 04:30:48', ['2016-02-26 04:30:48', '2016-10-29 04:30:48', '2016-10-29 04:30:47']),
```

Comment:
@ziafazal sorry for causing the confusion. Basically, my point is that the expected response is always '2016-10-29 04:30:48'. It would be good to have some different values for expected value.